ports/influxdb-cxx/vcpkg.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "influxdb-cxx",
+  "version": "0.6.6",
```

```suggestion
  "version-semver": "0.6.6",
```
It might be `version-semver `here? Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions

---

I don't know. @offa do you think you follow [semantic versioning](https://semver.org/#semantic-versioning-specification-semver)?

---

I try, but in doubt better don't rely on it.

---

Ok, then I would say we stay with `version` and don't use `version-semver`